src/routes/savoringYourChildModule/savoringYourChildModule.js

```diff
@@ -9,67 +9,108 @@ import Share from '../../components/Share'
 import FAQ from '../../components/FAQ'
 
 type Props = {
-  title: string,
-  path: string,
-  typeform: string,
-  faq: [],
-  testimonials: [],
-  user: Object,
-  typeformUserId: '',
+  title?: string,
+  path?: string,
+  typeform?: string,
+  faq?: [],
+  testimonials?: [],
+  user?: Object,
+  typeformUserId?: '',
 };
 
-const savoringYourChildSectionForm = ({
-  title, path, typeform, faq, testimonials, user, typeformUserId,
-}:
-Props) => (
-  <div>
-    <div className="container">
-      <div className="row">
-        <div className="col-md-2 col-xs-12">
-          <Share path={path} title={title} />
-        </div>
-        <div className="col-md-8 col-xs-12">
-          <div className="mainheading">
-            <h1 className="posttitle">{title}</h1>
-          </div>
-          <div>
-            <div>User typeform ID: {typeformUserId},</div>
-            <div>User ID {user._id},</div>
+class savoringYourChildSectionForm extends React.Component {
+  static defaultProps = {
+    title: '',
+    path: '',
+    typeform: '',
+    faq: '',
+    testimonials: '',
+    user: '',
+    typeformUserId: '',
+  }
+
+  state = {
+    urlPaths: [
+      '/savoring-your-child/peaceful-ending',
+      '/savoring-your-child/savoring-the-future',
+      '/savoring-your-child/reunion',
+      '/savoring-your-child/reengaging-the-future',
+      '/savoring-your-child/special-days',
+      '/savoring-your-child/relationship-consolidation',
+    ],
+  }
+
+  moduleRedirect = () => {
+    const toRemove = this.props.path
+    const index = this.state.urlPaths.indexOf(toRemove)
+    this.state.urlPaths.splice(index, 1)
```

Comment:
`splice` mutates the array, and state should never be mutated. read about it for up to 30 minutes, and then if you have Qs let me know. Open a new trello card for this.